lib/services/catalog/controllers/catalog-search-controller.js

```diff
@@ -10,8 +10,10 @@ class CatalogSearchController {
 	}
 
 	get(req, res, next) {
+		const theChannel = req.query.channel ? req.query.channel : req.identity.channel.id;
```

Comment:
updated: https://github.com/oddnetworks/oddworks/pull/132/commits/5609a11525968c8bdee55d3a54e670d96c92ed81


> On Aug 8, 2016, at 10:35 AM, Beard of War <notifications@github.com> wrote:
> 
> In lib/services/catalog/controllers/catalog-search-controller.js <https://github.com/oddnetworks/oddworks/pull/132#discussion_r73886161>:
> 
> > @@ -10,8 +10,10 @@ class CatalogSearchController {
> >  	}
> >  
> >  	get(req, res, next) {
> > +		const theChannel = req.query.channel ? req.query.channel : req.identity.channel.id;
> Every other get always uses req.identity.channel.id and its not overridable. So we should follow suite here and not allow to provide a channel via req.query.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/oddnetworks/oddworks/pull/132/files/34b8423a5c3322f3ef88b294b275d94cad2c6f06#r73886161>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABRcd7PeYtpvQh91Fg-qTDR50YJftVaqks5qdz7HgaJpZM4JdyCX>.
>